### ISSUE TITLE
CanvasView and clear_rect

### ISSUE THREAD

**Issue by JeremiasE** [martinRenou/ipycanvas#58]
Clearing a rectangle is not visible in previous views of the widget. To be more precise construct a canvas widget and show it:
```
from ipycanvas import Canvas  
canvas = Canvas(size=(200, 200))  
canvas.fill_rect(0,0,100,100)  
canvas  
```
Now clear a rectangle:
```
canvas.clear_rect(0,0,20,20)
```
This does not result in a change of the visible canvas widget.
Showing another instance of the widget by putting
```
canvas
```
in a new cell shows the cleared rectangle however.

Is this intentional?

It seems to be due to the fact that the views are updated by drawing the model canvas on them, but clearing a rectangle makes it transparent. Putting `this.clear()` before 
the call to `drawImage` in `updateCanvas` fixes this for me.

**Comment by martinRenou** [comment]
I am surprised I did not see this! Thanks a lot :)

Indeed calling `this.clear` before updating the view should fix this. Do you want to open a PR so you get credits for your fix?

**Comment by JeremiasE** [comment]
I opened #60. I was afraid that clearing the canvas would result in flickering animations but at least in the game of life example I didn't notice anything.

**Comment by martinRenou** [comment]
Thanks a lot! I was also afraid of the flickering at first but I suppose it will be fine.

**Comment by martinRenou** [comment]
Fixed by #60

**Comment by martinRenou** [comment]
`0.4.1` with your fix released on PyPi :) Coming soon on conda
